wp-includes/widgets/class-wp-widget-media.php

```diff
@@ -101,20 +102,25 @@ public function widget( $args, $instance ) {
 	 * @see WP_Widget::update()
 	 *
 	 * @param array $new_instance Values just sent to be saved.
-	 * @param array $old_instance Previously saved values from database.
+	 * @param array $instance     Previously saved values from database.
 	 * @return array Updated safe values to be saved.
 	 */
-	public function update( $new_instance, $old_instance ) {
-		$instance = $old_instance;
+	public function update( $new_instance, $instance ) {
 
 		// ID and title.
 		$instance['id']    = (int) $new_instance['id'];
 		$instance['title'] = sanitize_text_field( $new_instance['title'] );
 
-		// Everything else.
-		$instance['align'] = sanitize_text_field( $new_instance['align'] );
-		$instance['size']  = sanitize_text_field( $new_instance['size'] );
-		$instance['link']  = sanitize_text_field( $new_instance['link'] );
+		if ( in_array( $new_instance['align'], array( 'none', 'left', 'right', 'center' ) ) ) {
+			$instance['align'] = 'align' . $new_instance['align'];
```

Comment:
`.align*` does exactly that. Better even because themes have optimized styles for it.